x-pack/plugins/security_solution/public/explore/components/paginated_table/index.tsx

```diff
@@ -138,7 +138,7 @@ export interface Columns<T, U = T> {
   name: string | React.ReactNode;
   render?: (item: T, node: U) => React.ReactNode;
   sortable?: boolean | Func<T>;
-  truncateText?: boolean;
+  truncateText?: boolean | { lines: number };
```

Comment:
I would suggest that you create another value to hold this. `truncateLimit?:  { lines: number }`. Also `truncateText` should have been named to `shouldTruncateText`.

Reply:
Agreed, I'd leave line 141 as it is and add it as an optional number, and put a comment to exaplain like it means how many lines to keep if provided.

Reply:
Hey, thank you for the review.

`boolean | { lines: number }` Is the type already supported by `EuiTable`. I only updated the component type to match `Eui`.

https://eui.elastic.co/#/tabular-content/tables#table-layout
![Screenshot 2024-10-14 at 21 22 24](https://github.com/user-attachments/assets/507c004a-5683-4370-a6f3-e76f2985dc7d)


https://github.com/elastic/kibana/blob/entity-store-enrich-processor-for-rebase/node_modules/@elastic/eui/eui.d.ts#L2477